semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/package.scala

```diff
@@ -0,0 +1,17 @@
+package scala.meta.internal
+
+import java.nio.file._
+import scala.tools.asm._
+import scala.tools.asm.ClassReader._
+import scala.tools.asm.tree._
+
+package object metacp {
+  implicit class XtensionAsmPathOps(path: Path) {
+    def toClassNode: ClassNode = {
```

Comment:
Nitpick. I personally think `toX` extension methods should be pure data transformations and not do I/O. I think java.nio and scala.concurrent get this right by making unsafe/blocking operations static methods on objects like `Await.result` and `Files.readAllBytes`.